features/authentication.feature

```diff
@@ -332,3 +332,15 @@ Feature: OAuth authentication
     Then the output should contain "github.com username:"
     And the output should contain "missing user"
     And the file "../home/.config/hub" should not contain "user"
+    
+  Scenario: Config file is not writeable, should exit before asking for credentails
+      Given I set the environment variables to:
```

Comment:
Nice job discovering this step built into Aruba. However, since we're only testing one value, I think a Cucumber table is an overkill. There's a simpler step in our `steps.rb` in the format of:

```
Given $FOO is "bar"
```